andela/ah-frontend-valkyrie
# #162633478, #162633477  Implement Bookmarking Functionality

**What does this PR do?**

Implements Bookmark functionality feature in articles.

**Description of Task to be completed?**

- Create components for bookmarking and listing bookmarks.
- Create actions and reducers for bookmarks.
- Create tests for the feature.

**How should this be manually tested?**
- Run `npm install` to install dependencies.
- Run `npm test` to test the functionality.
- Run `npm start` to run the application.
- Check your redux dev tools to confirm that the state in your app appears in the dev tools.

**What are the relevant pivotal tracker stories?**

[#162633478](https://www.pivotaltracker.com/story/show/162633478)
[#162633477](https://www.pivotaltracker.com/story/show/162633477)
